src/nsupdate/main/templates/main/includes/tabbed_router_configuration.html

```diff
@@ -281,19 +281,29 @@ <h4>inadyn ({% trans "verified with" %} 1.99.11)</h4>
 
 # note: no IPv6 support in inadyn yet.
 </pre>
-        <h4>inadyn ({% trans "verified with" %} 2.1)</h4>
+        <h4>inadyn ({% trans "verified with" %} 2.10.0)</h4>
         <pre># /etc/inadyn.conf
 
 period       = 300
 
-provider ipv4@nsupdate.info {
+# use either ipv4 or ipv6 configuration below
+#provider ipv4@nsupdate.info {
+#    ssl      = true
+#    username = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
+#    password = {{ update_secret|default:"&lt;your secret&gt;" }}
+#    hostname = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
+#}
+
+# IPv6 supported in inadyn, but no pre-defined provider
+custom ipv6@nsupdate.info {
     ssl      = true
     username = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
     password = {{ update_secret|default:"&lt;your secret&gt;" }}
     hostname = {{ host.get_fqdn|default:"&lt;your hostname&gt;" }}
+    ddns-server = {{ WWW_IPV6_HOST }}
+    ddns-path = "/nic/update?hostname=%h&myip=%i"
```

Comment:
btw, did you check that inadyn gives the correct v6 address here? usually an interface has multiple v6 addrs.

Reply:
inadyn checks the IPv6 addresses associated to an interface and suppresses those tagged link local, temporary, or obsolete.